packages/mobile/src/Components/Chip.tsx

```diff
@@ -1,16 +1,16 @@
 import * as React from 'react';
-import { Chip as PaperChip, ChipProps, withTheme } from 'react-native-paper';
-import { IThemeShape } from 'Provider/IThemeShape';
+import { Chip as PaperChip, ChipProps, withTheme, Theme } from 'react-native-paper';
 
 export interface IChipProps extends Partial<ChipProps> {
-  theme: IThemeShape;
+  theme: Theme;
 }
 
```

Comment:
[THIS IS NOT A REQUEST FOR CHANGE]
Just a heads up, we'll probably have to bring this theme back, since we'll need to make certain adjustments on how the theme is injected on the `PaperChip` component, but it's okay for now.